dist/manifest.json

```diff
@@ -33,7 +33,10 @@
       "https://superuser.com/*",
       "https://askubuntu.com/*",
       "http://stackoverflow.com/*",
-      "https://stackoverflow.com/*"
+      "https://stackoverflow.com/*",
+      "http://package.elm-lang.org/*",
+      "https://docs.rs/*",
```

Comment:
This one works perfectly, if you want to update this PR with only this one, I could merge right away.

URL tested: https://docs.rs/geojson/0.8.0/geojson/